pyneuroml/plot/Plot.py

```diff
@@ -240,6 +249,41 @@ def generate_plot(
         logger.info("Saved image to %s of plot: %s" % (save_figure_to, title))
 
     if show_plot_already:
+        if animate:
+            d = 5
```

Comment:
I need to test this out more. I thought the idea was that if I set duration to 5000ms, the animation would be 5000ms, but I think you're saying that this is not the case?

Reply:
yes, that is what is happening with my current implementation. 
My calculations could be improved I think and it should be possible to atleast get as close as possible to set duration, let me try again.

Reply:
Don't worry about getting it exactly right---close enough will do and it can be tweaked later